core/state-db/src/noncanonical.rs

```diff
@@ -220,7 +220,7 @@ impl<BlockHash: Hash, Key: Hash> NonCanonicalOverlay<BlockHash, Key> {
 				commit.data.inserted = self.last_canonicalized_overlay.iter().map(|(k, v)| (k.clone(), v.clone())).collect();
 				commit.data.deleted = overlay.deleted;
 			} else {
-				// TODO: borrow checker won't allow us to split out mutable references
+				// FIXME: borrow checker won't allow us to split out mutable references
```

Comment:
```suggestion
				// borrow checker won't allow us to split out mutable references
```

Comment isn't referring to any change needed for this to happen, only explains why it is how it is.